app/controllers/contestant_projects_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ContestantProjectsController < ApplicationController
+  def create
+    project = Project.find(params[:id])
+    contestant = Contestant.find(params[:add_contestant])
+    contestant_project = ContestantProject.create!(
+      project_id: project.id,
+      contestant_id: contestant.id
+    )
+    redirect_to "/projects/#{project.id}"
```

Comment:
Nice job creating this controller for this action! It's exactly the setup I would do too. 👍